tv-shows/src/components/shared/Sidebar/SidebarNavigation.tsx

```diff
@@ -1,7 +1,21 @@
+"use client";
 import { Button, Flex, Box } from "@chakra-ui/react";
 import NextLink from "next/link";
+import { mutate } from "swr";
+import { swrKeys } from "../../../fetchers/swrKeys";
+import { useRouter } from "next/navigation";
 
 export const SidebarNavigation = () => {
+  const router = useRouter();
+
+  const handleLogout = () => {
+    localStorage.removeItem("access-token");
+    localStorage.removeItem("client");
+    localStorage.removeItem("uid");
+    localStorage.removeItem("expiry");
+    mutate(swrKeys.me, null, { revalidate: true });
```

Comment:
Mozemo zasad i ne revalidirati (nema potrebe da ide na api kad smo sigurni da ce sljedeci request failati jer smo pobrisali header-e i cache). Nek nam veruje i automatski nas "izlogira".
```suggestion

export const SidebarNavigation = () => {
  const router = useRouter();

  const handleLogout = () => {
    localStorage.removeItem("access-token");
    localStorage.removeItem("client");
    localStorage.removeItem("uid");
    localStorage.removeItem("expiry");
    mutate(swrKeys.me, null, { revalidate: false });
```